exporter/datadogexporter/internal/hostmetadata/internal/system/package_test.go

```diff
@@ -10,5 +10,7 @@ import (
 )
 
 func TestMain(m *testing.M) {
-	goleak.VerifyTestMain(m)
+	goleak.VerifyTestMain(m,
+		goleak.IgnoreAnyFunction("github.com/cihub/seelog.(*asyncLoopLogger).processQueue"),
```

Comment:
What is this? Can you add a tracking issue?

Reply:
Agent module is logging error here https://github.com/DataDog/datadog-agent/blob/78ca7aad3407f9202fc0ae3f81623907a9e32691/pkg/util/hostname/validate/validate.go#L40
I don't see a need to track it.

Reply:
@crobert-1 Could you help us with the best way to deal with this? (e.g. is there a template for tracking issues?)

Reply:
> I don't see a need to track it.

We aim to have no leaks at all in the Collector, this should be tracked and eventually fixed in the Agent.

Reply:
Created internal jira to track this work in Agent.

Reply:
Please add frequency and information to this issue: https://github.com/cihub/seelog/issues/182

Also, please add a reference in this file to that issue so that we have future context for why this ignore is necessary.